AUTHORS.md

```diff
@@ -0,0 +1,7 @@
+Authors
```

Comment:
I am sorry, but I don't want to maintain a `AUTHORS.md`, this could be looked up on github. Please remove the file.